app/views/site/_study_visualize.html.erb

```diff
@@ -5,7 +5,7 @@
   <div class="col-md-13" id="render-target">
     <div class="row-offcanvas row-offcanvas-right">
       <div id="view-options-nav"><a href="#view-options" id="view-option-link" data-toggle="offcanvas"><i class="fas fa-cog" aria-hidden="true"></i> View Options </a></div>
-      <ul class="nav nav-tabs" role="tablist" id="view-tabs">
+      <ul class="nav nav-tabs" role="tablist" id="view-tabs" data-tablist-name='Default'>
```

Comment:
```suggestion
      <ul class="nav nav-tabs" role="tablist" id="view-tabs" data-tablist-name='explore-default'>
```

Reply:
I kept it consistent with the "Explore - Default" e2597db2